externs/example.js

```diff
@@ -13,7 +13,12 @@ common.getRendererFromQueryString = function(opt_default) {};
 
 
 /**
+ * @param {function(new:ol.style.Style, olx.style.StyleOptions=)} Style Style.
```

Comment:
This textual description is a bit misleading, the argument is not a Style, but a Style constructor. here and for the other params... and totally minor